tests/helpers.js

```diff
@@ -30,4 +32,13 @@ module.exports = {
       },
     },
   },
+
+  findByClass: function(component, className){
+    return TestUtils.findRenderedDOMComponentWithClass(component, className);
```

Comment:
you could maybe use the ``fn.apply(arguments.slice())`` trick but I forget how it works.